walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc

```diff
@@ -54,61 +51,157 @@ image::images/arch.png[architecture, role="integr8ly-img-responsive"]
+
+=== Running DataSync client and server applications
+
+. Open new terminal window
+. Execute `docker-compose up -d` to start MongoDB container
+. Execute `yarn prepare:client` to build client side React application and include it in server runtime
+. Execute `yarn start:server`. Terminal should print similar output:
+
+----
+    ***********************************************************
+    🎮 Ionic PWA application available at http://localhost:4000/
+    🚀 GraphQL Playground is available at http://localhost:4000/graphql
+    ***********************************************************
+----
+
+. Open `PWA application URL` printed in terminal
+
+[type=verification]
+****
+. Check if website was loaded properly
+. Select + icon to create new item
+. On new screen put `name` and `description`
+The client should create a task and it should be
+. New task should appear on the list
+****
+
+[type=verificationFail]
+****
+Check the logs of the console
+Verify that you followed each step in the procedure above.
+If you are still having issues, contact your administrator.
+****
+
+=== Interacting with embeeded GraphQL Playground
+
+GraphQL Playground acts as GraphQL API client that allows
+you to interact with your types without implementing new views in your application.
+In this section we are going to focus on learning who to use playground.
+
+. Open new terminal window
+. Execute `yarn start:server`
+. Open GraphQL Playground URL printed in console.
+You can use the GraphQL playground to interact with the server API as described in the next step.
+. Go to the Playground interface and replace the text in the left pane of the screen with the following query and mutation:
 
-. Go to the link:{route-ionic-showcase-server-host}/graphql[GraphQL playground, window="_blank"].
-. Replace the text in the left pane of the screen with the following query and mutation:
 ----
 query listTasks {
   allTasks {
     title,
     description,
+    address,
     id
   }
 }
 
 mutation createTask {
-  createTask(title: "complete the walkthrough", description: "complete the GraphQL walkthrough") {
+  createTask(title: "complete the walkthrough", description: "complete the GraphQL walkthrough", address: "NA") {
     title,
     description,
     version,
+    address,
     id
   }
 }
 ----
-. Click the execute query/play button in the middle of the playground screen and then select *createTask* from the menu. The system should create a task.
-. Click the execute query/play button again and then then select *listTasks* from the menu.
 
 [type=verification]
 ****
-
-Look at the right side of your playground screen. Do you see the following code displayed?
-
+. Click the Run icon in the middle of the playground screen.
+. Choose createTask from the menu.
+The system should create a task.
+. Choose listTasks from the Run menu.
+. Check that the following is displayed in the right hand panel:
```

Comment:
This doesn't make sense to have two instructions 4. & 5.  I'm assuming both 4 & 5 refer to the json below?  Perhaps make the instruction like this '4. Check that the following is displayed in the right hand panel and that it includes the field we have added in the previous steps.'